python/shipPatRec.py

```diff
@@ -352,7 +352,7 @@ def fast_hough_pat_rec_stereo_views(SmearedHits_stereo, recognized_tracks_y, min
                 if ahit2['digiHit'] in used_hits:
                     continue
 
-                if abs(ahit1['zx_projection']) > 300 or abs(ahit2['zx_projection']) > 300:
+                if abs(ahit1['zx_projection']) > 200 or abs(ahit2['zx_projection']) > 200:
```

Comment:
Maybe good to define a const instead of repeating these two numbers everywhere. Please use explicit units for clarity.